Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -22,6 +22,22 @@ function Test-RegistrationComplete($authUrl)
     return $false
 }
 
+function Unlock-ConfigurationSections(){
+    $assembly = [System.Reflection.Assembly]::LoadFrom("$env:systemroot\system32\inetsrv\Microsoft.Web.Administration.dll")
+    $manager = new-object Microsoft.Web.Administration.ServerManager  
+    $config = $manager.GetApplicationHostConfiguration()
+    
+    $section = $config.GetSection("system.webServer/security/authentication/anonymousAuthentication")
+    $section.OverrideMode = "Allow"    
+    Write-Host "Unlocked system.webServer/security/authentication/anonymousAuthentication"
+
+    $section = $config.GetSection("system.webServer/security/authentication/windowsAuthentication")
+    $section.OverrideMode = "Allow"    
+    Write-Host "Unlocked system.webServer/security/authentication/windowsAuthentication"
+    
+    $manager.CommitChanges()
```

Comment:
What happens if the two sections are already unlocked, is an error thrown, or does it proceed silently? We would ultimately want to proceed silently since that wouldn't be a fatal error.

Reply:
if they are already unlocked it proceeds silently. no exception thrown